journal-of-international-economic-law.csl

```diff
@@ -0,0 +1,405 @@
+  <info>
+    <title>Journal of International Economic Law</title>
+    <id>http://www.zotero.org/styles/journal-of-international-economic-law</id>
+    <link href="http://www.zotero.org/styles/journal-of-international-economic-law" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/mcgill-en"/>
+    <link href="http://www.oxfordjournals.org/our_journals/jielaw/for_authors/stylesheet_2.pdf" rel="documentation"/>
+    <author>
+      <name>Catherine Brendow</name>
+      <email>catherine.brendow@graduateinstitute.ch</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <issn>1369-3034</issn>
+    <summary>According to the style sheet of the Journal of International Economic Law</summary>
+    <updated>2016-06-21T14:55:22+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <style-options punctuation-in-quote="false"/>
+    <terms>
+      <term name="et-al">et al</term>
+      <term name="ordinal">th</term>
+      <term name="ordinal-01">st</term>
+      <term name="ordinal-02">d</term>
+      <term name="ordinal-03">d</term>
+      <term name="ordinal-11">th</term>
+      <term name="ordinal-12">th</term>
+      <term name="ordinal-13">th</term>
+      <term name="paragraph" form="short">
+        <single>para</single>
+        <multiple>paras</multiple>
+      </term>
+      <term name="accessed">visited</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" delimiter-precedes-last="never"/>
+      <et-al term="et-al"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title" form="short" strip-periods="true"/>
+  </macro>
+  <macro name="author-bib">
+    <names variable="author">
+      <name and="text" name-as-sort-order="all"/>
+      <substitute>
+        <names variable="editor">
+          <name and="text" name-as-sort-order="all"/>
+        </names>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-note">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never"/>
+      <substitute>
+        <text macro="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="internet-location">
+    <choose>
+      <if variable="URL">
+        <text variable="URL" prefix=" "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="issued-long">
+    <text variable="volume"/>
+    <date delimiter=" " variable="issued">
+      <date-part name="day"/>
+      <date-part name="month" form="long"/>
+      <date-part name="year" form="long"/>
+    </date>
+  </macro>
+  <macro name="edition">
+    <number variable="edition" form="ordinal" text-case="lowercase" suffix=" "/>
```

Comment:
lowercase doesn't do anything here (it's also the default in English where it _does_ do anything). Remove, please.